bigquery_etl/dryrun.py

```diff
@@ -217,6 +217,11 @@ def is_valid(self):
         return True
 
 
+def sql_file_valid(sqlfile):
+    """Dry run SQL files."""
+    return DryRun(sqlfile).is_valid()
+
+
```

Comment:
I had to move this out of `main` otherwise it raised `AttributeError: Can't pickle local object 'main.<locals>.sql_file_valid'`